BrightEmah123/ADC-Quick-Credit
# #166017557 Build out endpoints for user signup and signin

**What does this PR do?**
Have api endpoints work for signup and signin

**Description of Task to be Completed**
Have the following endpoints working:

**POST**  api/v1/auth/signup
**POST** api/v1/auth/signin
**PATCH** api/v1/users/:email/verify

**How should this be manually tested**
After cloning the repo, cd to the repo on the local system and run command _npm run start:dev_
 

- Use Postman and test the endpoints stated above

 

- for the user verify endpoint, select an email from the models folder and input the mail to the _:email_ specification

**Where are the relevant pivotal tracker stories?**
 #166017557
